File: web/src/main/java/com/github/dkurata38/open_web_library/web/library/LibraryController.java
package com.github.dkurata38.open_web_library.web.library;

import lombok.RequiredArgsConstructor;
import org.springframework.stereotype.Controller;

@Controller
@RequiredArgsConstructor
public class LibraryController {
	private static final String resourcesPath = "libraries";
	private static final String resourcePath = "library/{id}";
	private static final String ownResourcePath = "library/{id}";

}

File: web/src/main/kotlin/com/github/dkurata38/open_web_library/web/library/LibraryController.kt
package com.github.dkurata38.open_web_library.web.library

import org.springframework.stereotype.Controller

@Controller
class LibraryController {
	companion object {
		private const val resourcesPath = "libraries"
		private const val resourcePath = "library/{id}"
		private const val ownResourcePath = "library/{id}"
	}

}
